iml-api/src/graphql.rs

```diff
@@ -467,6 +469,19 @@ impl QueryRoot {
 
         Ok(xs)
     }
+    /// List Stratagem reports
+    async fn stratagem_reports(
+        _context: &Context,
+    ) -> juniper::FieldResult<Vec<String>> {
+        let paths = tokio::fs::read_dir(get_report_path()).await?;
+
+        let items: Vec<String> = paths
+            .map_ok(|x| x.file_name().into_string().expect("DirEntry to string"))
```

Comment:
Seems like we should return the error instead of panicking here.

Reply:
I'm trying to think of the best way to do this. So far I came up with:
```
let items: Vec<String> = paths
            .map_ok(|x| {
                x.file_name().into_string().map_or_else(
                    |e| format!("Error getting stratagem report file name: {:?}", e),
                    |x| x,
                )
            })
            .try_collect()
            .await?;
```
However, I feel like I'm missing something..

Reply:
Pushed an update.